pom.xml

```diff
@@ -731,6 +777,7 @@
               <libDirectory>${nativeLibOnlyDir}</libDirectory>
               <configureArgs>
                 <configureArg>${extraConfigureArg}</configureArg>
+                <configureArg>${extraConfigureArg2}</configureArg>
```

Comment:
We need this because it doesn't split by space?

Reply:
Yeah